test/runtests.jl

```diff
@@ -7,5 +7,6 @@ using Test
   @time @testset "WeakDivPerpTests" begin include("WeakDivPerpTests.jl") end
   @time @testset "LaplaceBeltramiCubedSphereTests" begin include("LaplaceBeltramiCubedSphereTests.jl") end
   @time @testset "WaveEquationCubedSphereTests" begin include("WaveEquationCubedSphereTests.jl") end
+  @time @testset "Williamson2_ShallowWaterExplicit" begin include("Williamson2_ShallowWaterExplicit.jl") end
```

Comment:
Remove underscore from test set and file name. Add the postfix `Tests` at the end to both names.